.github/workflows/tests.yml

```diff
@@ -155,7 +156,9 @@ jobs:
           git diff --exit-code
       - name: Check docs
         run: |
+          # TODO: stardoc doesn't currently support bzlmod: https://github.com/bazelbuild/stardoc/issues/117
```

Comment:
I'll create a separate PR for this it will update the version to the required stardoc (requires some changes to our use of `bzl_library`)